setup.md

```diff
@@ -8,72 +8,156 @@ permalink: /setup/
+ </tr>
+ <tr>
+  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Clone the Flutter repo</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+  <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Update your path</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#get-the-flutter-sdk">Run <code>flutter doctor</code> and resolve any issues found</a></td><td align="center">Y</td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#editor-setup">Install and configure an IntelliJ IDE to develop Flutter apps</a></td><td align="center"></td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#ios-setup">Perform iOS setup tasks to develop apps for iOS</a></td><td align="center"></td>
+ </tr>
+ <tr>
+   <td align="center">&#10004;</td><td><a href="#android-setup">Perform Android setup tasks to develop apps for Android</a></td><td align="center"></td>
+ </tr>
+</table>
+
+Using an IDE is optional. However, we recommend using an IntelliJ IDE with Flutter for code completion, 
+inline error checking, and visual debugging features.
+{: .alert-warning}
 
 ## System requirements
 
 To install and run Flutter, your development environment must meet these minimum requirements:
 
 * **Operating Systems**: Mac or Linux (64-bit). Windows support is [in progress](https://github.com/flutter/flutter/issues/138).
+* **Disk Space**: 710 MB on Mac, 558 MB on Linux (does not include disk space for Xcode or Android Studio).
 * **Tools**: Flutter depends on these command-line tools being available on your environment.
   * `bash`, `mkdir`, `rm`, `git`, `curl`, `unzip`
 
 ## Get the Flutter SDK
 
-To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
+1. To get Flutter, use `git` to clone the repository and then add the `flutter` tool to your path:
 
-<pre>
-$ git clone https://github.com/flutter/flutter.git
-$ export PATH=`pwd`/flutter/bin:$PATH
-</pre>
+   ```
+   $ git clone https://github.com/flutter/flutter.git
+   $ export PATH=`pwd`/flutter/bin:$PATH
+   ```
 
-Run the following command to see if there are any dependencies you need to install to complete
+   where `pwd` is the current working directory, for example:
+   ```
+   $ export PATH=/Users/obiwan/flutter/bin:$PATH
+   ```
+
+   The above command sets your PATH variable temporarily, for the current terminal window.
+   The steps for modifying this variable permanently for all terminal sessions are machine-
+   specific. Typically you add a line to a file that is executed whenever you open 
+   a new window:
+
+   ```
+   export PATH=~/flutter/bin
+   ```
+and then run the `source <filename>` command to refresh the window. For example on Linux, 
+you might edit and source `~/.bash_profile`. On macOS, it might be `/etc/bashrc`.
```

Comment:
There was a reason it was worded this way - on a loaner Mac it was necessary to edit /etc/bashrc to get the PATH to stick. It took a lot of trying and time. Despite that, I'll make this more general...

FYI it's OK (and totally expected) if your mileage varies on this one. The point is YMMV and we can't tell you exactly which file in what directory; it depends how the user's machine is set up. We're not actually pointing to any particular file.